<a href="https://colab.research.google.com/github/LuisaMariaMontoyaConde1994/ESTADISTICAII/blob/main/Actividad_binomial.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

**Paula Geraldine Morales Díaz**

# **Actividad**
En una elección, un candidato tiene una probabilidad del $60\%$ de recibir un voto en cada votación individual. Si se emiten $20$ votos.


1.   ¿Cuál es la probabilidad que que al menos el candidato obtenga $12$ votos?
2.   ¿Cuál es la probabilidad  que a lo más $10$ votos?
3.   ¿Cuál es la probabilidad  que obtenga entre $9$ y $16$ votos?
4.   ¿Cuál es la probabilidad  que obtenga $15$ votos exactamente?

*   $p=0.6$
*   $q=1-0.6=0.4$
*   $n=20$



In [1]:
import numpy as np
import matplotlib.pyplot as plt
from scipy.stats import binom

¿Cuál es la probabilidad que que al menos el candidato obtenga $12$ votos?
$$(X\geq 12)$$

In [2]:
n=20
p=0.6
x=12
prob_1=1-binom.cdf(x,n,p)
print(prob_1)

0.41589293755753554


**La probabilidad de que al menos el candidato obtenga $12$ votos es del $41.58\%$**

¿Cuál es la probabilidad  que a lo más $10$ votos?
$$(X\leq 10)$$

In [3]:
n=20
p=0.6
x=10
prob_1=binom.cdf(x,n,p)
print(prob_1)

0.24466279668360685


**La probabilidad de que a lo más el candidato obtenga $10$ votos es del $24.46\%$**



 ¿Cuál es la probabilidad  que obtenga entre $9$ y $16$ votos?
 $$P(9\leq X \leq 16)$$


In [4]:
n=20
p=0.6
x_1=9
x_2=16
prob=binom.cdf(x_2,n,p)-binom.cdf(x_1,n,p)
print(prob)

0.856517591062775


**La probabilidad de que el candidato obtenga entre $9$ y $16$ votos es del $85.65\%$**

¿Cuál es la probabilidad  que obtenga $15$ votos exactamente?
$$P(x=15)$$

In [5]:
n=20
p=0.6
x=15
pro=binom.pmf(x, n, p)
print(pro)

0.07464701952887111


**La probabilidad de que el candidato obtenga exactamente $15$ votos es del $7.4\%$**